modules/insights-operator-about.adoc

```diff
@@ -8,12 +8,12 @@
 
 The Insights Operator periodically gathers configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry.
 
-Users of {product-title} can display the report of each cluster in {cluster-manager-url}. If any issues have been identified, Insights provides further details and, if available, steps on how to solve a problem.
+Users of {product-title} can display the report of each cluster in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]. If any issues have been identified, Insights provides further details and, if available, steps on how to solve a problem.
 
 The Insights Operator does not collect identifying information, such as user names, passwords, or certificates. See link:https://console.redhat.com/security/insights[Red Hat Insights Data & Application Security] for information about Red Hat Insights data collection and controls.
 
 Red Hat uses all connected cluster information to:
 
-* Proactively identify potential cluster issues and provide a solution and preventive actions in {cluster-manager-url}
+* Identify potential cluster issues and provide a solution and preventive actions in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]
```

Comment:
The same applies here, you could consider replacing "link:[https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]" with "{cluster-manager-url}" for consistency with this docs library.

Reply:
see above